task/sast-unicode-check/0.1/sast-unicode-check.yaml

```diff
@@ -0,0 +1,269 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "konflux"
+  name: sast-unicode-check
+spec:
+  description: >-
+    Scans source code for non-printable unicode characters in all text files.
+  results:
+    - description: Tekton task test output.
+      name: TEST_OUTPUT
+  params:
+    - name: image-url
+      type: string
+      description: Image URL.
+      default: ""
+    - name: image-digest
```

Comment:
Why does the task need the `image-digest` parameter?

Reply:
Yes, It should be removed, I don't know why it exists; I think it exists in the original version